core/CMakeLists.txt

```diff
@@ -154,6 +154,21 @@ if(THEROCK_ENABLE_HIP_RUNTIME)
       libhiprtc-builtins.so
       libhiprtc.so
   )
+  therock_cmake_subproject_declare(hip-tests
+    USE_DIST_AMDGPU_TARGETS
+    EXTERNAL_SOURCE_DIR "${THEROCK_ROCM_SYSTEMS_SOURCE_DIR}/projects/hip-tests/catch"
```

Comment:
Does this work as-is? I think @geomin12 tried something similar and saw conflicts between the forked version of Catch2 in hip-tests (https://github.com/ROCm/rocm-systems/tree/develop/projects/hip-tests/catch/external/Catch2) and the version we bundle in TheRock: https://github.com/ROCm/TheRock/blob/5c520ac6c4950e408e5cd48179b0c328a259e015/third-party/Catch2/CMakeLists.txt#L1-L14